flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -883,8 +883,6 @@ private void onTriggerFailure(@Nullable PendingCheckpoint checkpoint, Throwable
                 synchronized (lock) {
                     abortPendingCheckpoint(checkpoint, cause);
                 }
-            } else {
-                LOG.warn("Failed to trigger checkpoint for job {}.)", job, throwable);
```

Comment:
nit: Why can't we just replace `warn` with info and log `throwable.getMessage()` instead of `throwable`?
So that exception isn't swallowed (in cases other than NOT_ALL_TASK_RUNNING) and logginng isn't duplicated.

Is it to preserve the current (1.12) behavior?

Reply:
Very thanks for the review! The initial thought is in case in the future we might throws other exceptions that need to be logged fully, so we may deal with each exception log separately. But it indeed seems better to not swallow exceptions, thus I change the PR to be modify the log level directly.